### ISSUE TITLE
Address Field not populated when licensee is using a PO Box

### ISSUE THREAD

**Issue by ckinsler** [k3ng/hamdb#1]
If a licensee is using a PO Box on their FCC record, their street address (position 16) in EN.dat is typically empty.  

You may wish to edit your EN temporary file creation section to utilize the PO Box (position 20) during the import if that is the case, so something shows during the subsequent load and query.  

Also note that the PO Box is just the number in the FCC file, so it probably also makes sense to prefix with "PO Box " before import.  Something like:

```
     if [ ! -f en_temp.txt ]; then
      echo "Creating temporary file for table en..."
      cat ./EN.dat | sed s/[\\\"]//g | awk -F "|" '{
        address = $16;
        pobox = $20;
        if (address1== "") {
            address = "PO Box " pobox;
        }
        print "\"" $2 "\",\"" $5 "\",\"" $8 "\",\"" $9 "\",\"" $10 "\",\"" $11 "\",\"" address "\",\"" $17 "\",\"" $18 "\",\"" $19 "\""
    }' > ./en_temp.txt
    fi
```

**Comment by k3ng** [comment]
Hi Chris.  Thank you for catching this and making a coding suggestion.  I am going to test and implement.  Thanks!

**Comment by k3ng** [comment]
Code implemented in version 2024.12.09.1500.  Thanks, Chris!!!
